gotocompany/optimus/core/v1beta1/resource.proto

```diff
@@ -175,6 +176,12 @@ message ResourceStatus {
   string message = 3;
 }
 
+message Deprecation {
+  string reason = 1;
+  google.protobuf.Timestamp date = 2;
```

Comment:
should we use this proto https://github.com/googleapis/googleapis/blob/master/google/type/date.proto for date only case ?